cms/static/cms/js/modules/cms.plugins.js

```diff
@@ -327,17 +327,22 @@ var CMS = window.CMS || {};
                 }
 
                 var bounds = this.options.plugin_restriction;
-                var type = clipboardPlugin.data('settings').plugin_type;
-                var parent_bounds = $.grep(clipboardPlugin.data('settings').plugin_parent_restriction, function (r) {
-                    // special case when PlaceholderPlugin has a parent restriction named "0"
-                    return r !== '0';
-                });
-                var currentPluginType = this.options.plugin_type;
+                if (clipboardPlugin && clipboardPlugin.data('settings')) {
```

Comment:
can just check for `clipboardPlugin.data('settings')`, clipboardPlugin is always truthy since it's a jquery object